content/docs/en/elements/layouts/grid-layout.md

```diff
@@ -51,23 +55,43 @@ The GridLayout is a layout that arranges its child elements in a table structure
 ```
 <img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout_fixed_auto.svg" />
 
+#### Grid layout with complex sizing and merged cells
+
+The following example creates a complex grid with responsive design, mixed width and height settings, and some merged cells.
+
+```html
+<GridLayout columns="40, auto, *" rows="40, auto, *" backgroundColor="#3c495e">
+  <Label text="0,0" row="0" col="0" backgroundColor="#43b883"/>
+  <Label text="0,1" row="0" col="1" colSpan="2" backgroundColor="#1c6b48"/>
+  <Label text="1,0" row="1" col="0" rowSpan="2" backgroundColor="#289062"/>
+  <Label text="1,1" row="1" col="1" backgroundColor="#43b883"/>
+  <Label text="1,2" row="1" col="2" backgroundColor="#289062"/>
+  <Label text="2,1" row="2" col="1" backgroundColor="#1c6b48"/>
+  <Label text="2,2" row="2" col="2" backgroundColor="#43b883"/>
+</GridLayout>
+```
+<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout.svg" />
```

Comment:
You can change it to `https://art.nativescript-vue.org/layouts/grid_layout_complex.svg`